Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -322,6 +347,8 @@ class EditorTestSuite:
     use_null_renderer = True
     # Maximum time for a single editor to stay open on a shared test
     timeout_editor_shared_test = 300
+    # Marks the test as a MaterialEditor test instead of an Editor test.
+    material_editor_test = False
```

Comment:
Instead of a single bool flag, should change it to be more flexible for the future? Are there other types of launchers (i.e. game launcher) that we may want to use this for?  Perhaps it should be a parameterization?

Reply:
my advise is to use python Enum class to define what options are supported and default to the editor.exe option
from enum import Enum
class Launcher(Enum):
    EDITOR = 1
    MATERIAL_EDITOR = 2
this parameter can be specified as Launcher.MATERIAL_EDITOR
this provides for expansion when we add more tools and launchers. Names can be whatever works these are just an example.